shark/shark_runner.py

```diff
@@ -84,12 +85,25 @@ def __init__(
         self.shark_runner = SharkRunner(self.model, self.input, dynamic,
                                         self.device, jit_trace, from_aot)
 
-    def forward(self, inputs):
+    def debug_forward(self, inputs):
         # TODO Capture weights and inputs in case of AOT, Also rework the
         # forward pass.
+        begin = time.time()
         inputs = self.input if self.from_aot else inputs
         input_list = [x.detach().numpy() for x in inputs]
-        return self.shark_runner.forward(input_list)
+        out = self.shark_runner.forward(input_list)
+        end = time.time()
+        return out, end - begin
+
+    def forward(self, inputs):
```

Comment:
The forward function shouldn't change or have any timings. Just create debug_forward same as forward with timings.

Reply:
benchmark_forward instead of debug_forward

Reply:
And the models by default will call benchmark_forward?

Reply:
yeah since they are samples that is ok.  We can add a comment saying switch to forward() if using in production.